aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/src/main/java/com/azure/spring/sample/aad/security/WebSecurityConfig.java

```diff
@@ -30,8 +29,7 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                 .anyRequest().permitAll()
                 .and()
             .csrf()
-                .csrfTokenRepository(CookieCsrfTokenRepository.withHttpOnlyFalse())
-                .and()
+                .disable()
```

Comment:
Do we have an issue to link here?

Reply:
To explain why we are disabling it.

Reply:
Opened https://github.com/Azure/azure-sdk-for-java/issues/32484

Reply:
@moarychan, I don't see why we decided to disable this in the issue you created. Should we add some comments in your code here or add some explanation on the issue you created?

Reply:
Sorry, updated it.